mysql-test/t/cte_recursive.test

```diff
@@ -1304,6 +1304,7 @@ drop table folks;
 create table t1(a int);
 insert into t1 values (0),(1),(2),(3),(4),(5),(6),(7),(8),(9);
  
+--source include/analyze-format.inc
```

Comment:
The failure in cte_recursive happens, not here, but in cte_recursive.test:1514 (which corresponds to cte_recursive.result:2367).